src/frontend/screens/Track/MapPreview.tsx

```diff
@@ -0,0 +1,118 @@
+import React, {FC} from 'react';
+import {StyleSheet} from 'react-native';
+import MapboxGL from '@rnmapbox/maps';
+import {LocationHistoryPoint} from '../../sharedTypes/location.ts';
+import Mapbox from '@rnmapbox/maps';
+import {TrackMapLayer} from '../../sharedComponents/TrackMapLayer.tsx';
+import {convertObservationsToFeatures} from '../../lib/utils.ts';
+import {Observation} from '@mapeo/schema';
+import {BLACK} from '../../lib/styles.ts';
+import {useHeaderHeight} from '@react-navigation/elements';
+interface TrackScreenMapPreview {
+  locationHistory: LocationHistoryPoint[];
+  observations: Observation[];
+}
+
+const MAP_STYLE = Mapbox.StyleURL.Outdoors;
+
+export const MapPreview: FC<TrackScreenMapPreview> = ({
+  locationHistory,
+  observations,
+}) => {
+  const [swBoundary, neBoundary] = getAdjustedBounds(locationHistory);
+  const headerHeight = useHeaderHeight();
+
+  return (
+    <MapboxGL.MapView
+      style={styles.map}
+      zoomEnabled={false}
+      logoEnabled={false}
+      scrollEnabled={false}
+      pitchEnabled={false}
+      rotateEnabled={false}
+      compassEnabled={false}
+      scaleBarEnabled={false}
+      styleURL={MAP_STYLE}>
+      <MapboxGL.Camera
+        animationMode="none"
+        padding={{
+          // with a transparent header, we want to make sure the title does not sit on top of the track by using this header height.
+          paddingTop: headerHeight + 25,
+          paddingRight: 25,
+          paddingLeft: 25,
+          paddingBottom: 25,
+        }}
+        bounds={{
+          ne: neBoundary!,
+          sw: swBoundary!,
+        }}
```

Comment:
nit: extract these to constants

Reply:
https://github.com/digidem/comapeo-mobile/pull/412/commits/ec523495b19e5212d2c9ad78e55cf87ef7de6450